Test/WebTestCase.php

```diff
@@ -574,18 +574,28 @@ public function loadFixtureFiles(array $paths = array(), $append = false, $omNam
 
         $files = $this->locateResources($paths);
 
-        // Check if the Hautelook AliceBundle is registered and if yes, use it instead of Nelmio Alice
-        $hautelookLoaderServiceName = 'hautelook_alice.fixtures.loader';
-        if ($container->has($hautelookLoaderServiceName)) {
-            $loaderService = $container->get($hautelookLoaderServiceName);
-            $persisterClass = class_exists('Nelmio\Alice\ORM\Doctrine') ?
-                'Nelmio\Alice\ORM\Doctrine' :
-                'Nelmio\Alice\Persister\Doctrine';
+        if (class_exists('Nelmio\Alice\Fixtures')) {
+            // nelmio/alice v2.*.*
+            // Check if the Hautelook AliceBundle is registered and if yes, use it instead of Nelmio Alice
+            $hautelookLoaderServiceName = 'hautelook_alice.fixtures.loader';
+            if ($container->has($hautelookLoaderServiceName)) {
+                $loaderService = $container->get($hautelookLoaderServiceName);
+                $persisterClass = class_exists('Nelmio\Alice\ORM\Doctrine') ?
+                    'Nelmio\Alice\ORM\Doctrine' :
+                    'Nelmio\Alice\Persister\Doctrine';
+
+                return $loaderService->load(new $persisterClass($om), $files);
+            }
 
-            return $loaderService->load(new $persisterClass($om), $files);
+            return Fixtures::load($files, $om);
+        } else {
+            $fidryLoaderServiceName = 'fidry_alice_data_fixtures.doctrine.persister_loader';
```

Comment:
why is `hautelook/alice-bundle 2.x` required? IMO `theofidry/alice-data-fixtures` should be enough for loading specific files

Reply:
coming back of this one: it actually depends of the features you want to provide. If you just want to load specific files, `theofidry/alice-data-fixtures` is enough. If you want the fixtures to be loaded from auto-discovered files, then HautelookAliceBundle is needed.